.github/workflows/tests.yml

```diff
@@ -79,9 +79,32 @@ jobs:
   #####################
   ##### RUBY TESTS ####
   #####################
+  ruby-test-files:
+    if: github.repository == 'exercism/website'
+
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.test-files.outputs.matrix }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - id: test-files
+        env:
+          FILES_PER_BATCH: 60
+        run: |
+          tests=$(find test -name *_test.rb -not -path system | xargs -n ${{ env.FILES_PER_BATCH }} | xargs -I {} echo '"{}"' | tr '\n' ',')
```

Comment:
From looking at the logs, system isn't getting excluded properly, which is why this is slow.

Reply:
(e.g. https://github.com/exercism/website/runs/4395693256?check_suite_focus=true)

Reply:
Ah, feck. It worked locally. I'll figure it out

Reply:
Should be fixed now